src/libdrakvuf/private.h

```diff
@@ -245,6 +245,8 @@ struct drakvuf
     int event_fd_cnt;          // auto-generated for poll()
     fd_info_t fd_info_lookup;  // auto-generated for fast drakvuf_loop lookups
     int poll_rc;
+
+    ipt_state_t ipt_state[16]; // TODO dynamically allocate
```

Comment:
:thinking: I've forgot about this, but anyway there is a cap on 16 vCPUs somewhere else:

https://github.com/tklengyel/drakvuf/blob/master/src/libdrakvuf/vmi.c#L1516-L1517

are we OK with leaving static array for now?

Reply:
Yea, we have a 16-vcpu limit right now

Reply:
I've removed this `//TODO` for now and added bound check whether `vcpu < 16`